_overviews/toolkit/munit-resources.md

```diff
@@ -80,6 +76,16 @@ using2TempFiles.test("merge tow files") {
 }
 ```
 {% endtab %}
+{% tab 'Scala 3' %}
+```scala
+val using2TempFiles: FunFixture[(os.Path, os.Path)] =
+  FunFixture.map2(usingTempFile, usingTempFile)
+
+using2TempFiles.test("merge tow files"):
+  case (file1, file2) =>
```

Comment:
@julienrf @bishabosha Should we use the fewer braces syntax in Scala 3 tabs?

Reply:
It will be available by default in 3.3.0, which is the LTS, so I am tempted to say yes.

BTW here you could also remove the `case` keyword:

~~~ scala
using2TempFiles.test("merge two files"):
  (file1, file2) =>
~~~